interfaces/cython/cantera/onedim.pyx

```diff
@@ -376,6 +399,11 @@ cdef class ReactingSurface1D(Boundary1D):
         self.surf = new CxxReactingSurf1D()
         self.boundary = <CxxBdry1D*>(self.surf)
 
+    def __init__(self, *args, **kwargs):
+        self._weakref_proxy2 = _WeakrefProxy()
```

Comment:
I think you can just make multiple weakrefs to the original `_weakref_proxy` object.